ngx_http_pta_module.c

```diff
@@ -170,6 +174,25 @@ ngx_http_pta_parse_cookie_header (ngx_array_t * headers, ngx_str_t * name,
 
           start = h[i]->value.data;
           end = h[i]->value.data + h[i]->value.len;
+#else
+    ngx_table_elt_t *headers, *h;
+
+    headers = r->headers_in.cookie;
+
+    for (h = headers, i = 0; h; h = headers->next, i++)
```

Comment:
use h != NULL

Reply:
@kuroishi Thanks for reviewing. I had fixed it.